pkg/network/node/ovscontroller.go

```diff
@@ -193,7 +193,7 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	// Table 80: IP policy enforcement; mostly managed by the osdnPolicy
 	otx.AddFlow("table=80, priority=300, ip, nw_src=%s/32, actions=output:NXM_NX_REG2[]", localSubnetGateway)
 	// eg, "table=80, priority=100, reg0=${tenant_id}, reg1=${tenant_id}, actions=output:NXM_NX_REG2[]"
-	otx.AddFlow("table=80, priority=0, actions=drop")
+	otx.AddFlow("table=82, priority=0, actions=drop")
```

Comment:
There is a comment on ovscontroller.go for every table. I  expect a comment for tables 81 and 82 too. Even if flows for for table 81 aren't created at all in this file.